src/sql/azdata.d.ts

```diff
@@ -3747,6 +3747,11 @@ declare module 'azdata' {
 			 */
 			customButtons: Button[];
 
+			/**
+			 * Any additional buttons that should be displayed for all pages on the left side of dialog footer.
+			 */
+			leftSideButtons: Button[];
```

Comment:
>leftSideButtons: Button[]; [](start = 3, length = 26)

Instead of this why not have a "position" field on the customButtons that says whether it's on the right/left?